crypto/keys/ed25519/ed25519.go

```diff
@@ -175,7 +176,7 @@ func (pubKey *PubKey) VerifySignature(msg []byte, sig []byte) bool {
 		return false
 	}
 
-	return ed25519.Verify(pubKey.Key, msg, sig)
+	return ed25519consensus.Verify(pubKey.Key, msg, sig)
```

Comment:
mind adding a comment here for future reference?

Reply:
added comment